cloud/monitoring/basic-objects/commands.md

```diff
@@ -53,7 +53,16 @@ For security reasons, Centreon Cloud has a built-in whitelist that defines which
 
 1. Log in as **root** to the poller that will run the commmand.
 2. Edit the following file: **/etc/centreon-engine-whitelist/my-whitelist.yml**.
-3. Add a line for your command, using a regex. Example: to authorize all commands from Centreon plugins, the correct syntax is **\/usr\/lib\/centreon\/plugins\/centreon_.*\.pl .***
+3. Use a regex to define which commands to authorize. Example:
+
+   ```yaml /etc/centreon-engine-whitelist/my_whitelist.yml
+   whitelist:
+     regex:
+       - \/opt\/my_plugins\/my_custom_plugin\.py .*
+    ```
+
+Use `.*` to include all arguments in the regex.
```

Comment:
```suggestion
The `.*`  at the end of the regex allows it to handle any arguments it may contain. Bear in mind that the format must be strictly indentical to the one above (including indents).
```